scripts/update-metadata.mjs

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from 'node:url';
+
+const CROWDIN_PROJECT_ID = process.env.CROWDIN_PROJECT_ID;
+const CROWDIN_PERSONAL_TOKEN = process.env.CROWDIN_PERSONAL_TOKEN;
+
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
Don't use __filename or __dirname, those are globals in Node and could get confusing